Add using directives to C# module tutorial

damonbarry opened the pull request:
The C# module tutorial makes use of 3 types whose namespaces aren't already referenced by the `aziotedgemodule` template code. So if you follow the tutorial, paste all the code into your own project as instructed, and build, you'll get compiler errors. I've added a step that instructs the reader to add the right namespace `using` directives.

azurecla commented:
Hi __@vidarmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

damonbarry committed:
Add using directives to C# module tutorial

The C# module tutorial makes use of 3 types whose namespaces aren't already referenced by the `aziotedgemodule` template code. So if you follow the tutorial, paste all the code into your own project as instructed, and build, you'll get compiler errors. I've added a step that instructs the reader to add the right namespace `using` directives.
